arch/arm/core/cortex_m/mpu/arm_mpu.c

```diff
@@ -95,6 +96,9 @@ static inline int _get_region_attr_by_type(arm_mpu_region_attr_t *p_attr,
 #ifdef CONFIG_NOCACHE_MEMORY
 	case NOCACHE_MEMORY_REGION:
 		_get_mpu_ram_nocache_region_attr(p_attr, P_RW_U_NA, base, size);
+#ifdef CONFIG_RAM_FUNCTION
+	case RAM_FUNCTION_REGION:
+		_get_mpu_ramfunc_region_attr(p_attr, base, size);
 		return 0;
```

Comment:
It looks like there is an issue there with a missing #endif. Probably a rebase issue.

Reply:
Thanks! Fixed.